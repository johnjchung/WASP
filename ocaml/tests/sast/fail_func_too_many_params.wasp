Func int twoArgs(int a, int b) {
    return a + b;
}

Endpoint badCitizen(GET:) {
    int sum = twoArgs(2, 3, 4);
}
