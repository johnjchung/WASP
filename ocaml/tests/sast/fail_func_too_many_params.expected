Fatal error: exception Semantic_check.FunctionArgTypeMismatchError("ARG TYPE MISMATCH ERROR ON FUNCTION CALL TO FUNCTION twoArgs")
